### ISSUE TITLE
add this to nuget

### ISSUE THREAD

**Issue by jrwren** [jrwren/petname-sharp#1]
Other petname lists in js and python are on npm and pip, it would be nice if a .net one were in nuget.

**Comment by mykeels** [comment]
I think you'd have to add this package to Nuget yourself, as its creator @jrwren ... If someone else does, they'd be responsible for maintain it and releasing updates. [Here's](https://medium.com/@burcakbas/nuget-101-creating-and-consuming-a-nuget-package-8fcf75db2e35) a good place to start ... [here's](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio-net-framework) another.

**Comment by jrwren** [comment]
Bummer. I don't have Windows or Visual Studio to follow these instructions.

**Comment by mykeels** [comment]
What do you build with? @jrwren Mac? Linux?

**Comment by jrwren** [comment]
Yes, I use both of those. I see in your first link that there is a `What about using dotnet cli?` 1/3 of the way down. I'll give that a try.

**Comment by jrwren** [comment]
I did something, but I don't know if it is right. Description and license didn't seem to show. https://www.nuget.org/packages/Petnames/1.0.0

https://github.com/jrwren/petname-sharp/blob/master/Petnames/Petnames.nuspec I don't know what to fill in, the docs are pretty poor IMO. They describe what each is, but don't tell you what you should change to make it look nice in the nuget UI

**Comment by mykeels** [comment]
I get a 404 when I visit https://www.nuget.org/packages/Petnames/1.0.0 ... Do you get content? Perhaps it's yet to propagate. It takes about 10 minutes.

**Comment by wilsonvargas** [comment]
I send a PR solved this. 🐱‍👤

Now is not necessary a nuspec file to generate a nuget package, that info is adding to .csproj file and when you build it creates a nuget package automatically.

Please check this [commit](https://github.com/jrwren/petname-sharp/pull/2/commits/40661e1f638e06eb30c22ec19f12c5c8c59e473a) to undestand it.

**Comment by jrwren** [comment]
Huge thanks. I really appreciate it.
